package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1051,6 +1051,8 @@ def _write_timestep(self, ts, multiframe=False):
            Writing now only uses the contents of the elements attribute
            instead of guessing by default. If the elements are missing,
            empty records are written out (Issue #2423).
+           Atoms are now checked for a chainID instead of being overwritten
+           by the last letter of the SegmentID (Issue #3144).
```

Comment:
```suggestion
           by the last letter of the `segid` (Issue #3144).
```

Try to use the actual names of the attributes.